src/transformState.js

```diff
@@ -5,7 +5,34 @@
  * @param {Object[]} actions
  */
 function transformState(state, actions) {
-  // write code here
+  for (const operation of actions) {
+    switch (true) {
+      case operation.type === 'addProperties': {
+        Object.assign(state, operation.extraData);
+        break;
+      }
+
+      case operation.type === 'removeProperties': {
+        for (const property of operation.keysToRemove) {
```

Comment:
I'd rather name it `key`
```suggestion
        for (const key of operation.keysToRemove) {
```